### ISSUE TITLE
Document how to consume ELN composes

### ISSUE THREAD

**Issue by davide125** [fedora-eln/eln#33]
Composes for ELN don't seem to be shipped to the mirror network, so they need to be fetched from https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/

This makes it a bit tricky to find a safe way to maintain a local mirror of a copy of the latest compose (e.g. for testing or CI purposes), so it'd be great to document that. Specifically:
- how to make sure the "latest" compose is actually finished and fit for consumption
- whether it's acceptable to just `wget -r` from odcs or if there's better ways (e.g. rsync, which doesn't seem to be exposed on that host though)
- how often it would be ok to do something like that (in my case, it'd probably be once a week)

Alternatively, if these could be added to the mirror network akin to the Rawhide composes that would make things easier.

Thanks!

**Comment by praiskup** [comment]
Having mirroring would be helpful.  Copr build failures are very frequent in ELN repositories, compared to others that build from mirrors.  E.g.: 
https://download.copr.fedorainfracloud.org/results/pemensik/bind-9.16/fedora-eln-x86_64/02100466-bind/builder-live.log.gz

**Comment by davide125** [comment]
Discussed in today's meeting, need to sort out #39 first, then revisit https://pagure.io/fedora-infrastructure/issue/9730

**Comment by praiskup** [comment]
Is there ETA for this?  I'm asking because we would like to release new mock-core-configs with mirrored
ELN repos soon (2-4 weeks).

**Comment by davide125** [comment]
`rsync://dl.fedoraproject.org/fedora-eln` is now available, which makes it possible to mirror ELN efficiently. Facebook has an mirror up at https://mirror.facebook.net/fedora-eln/ but I'm not aware of any others. Still need to update the documentation.

**Comment by praiskup** [comment]
Does it make sense to configure mirror manager even though there is only one mirror?

**Comment by praiskup** [comment]
Gently ping, any progress on ELN mirroring?

**Comment by sgallagher** [comment]
Currently there's still no effort underway to set up mirror manager for ELN. The Facebook mirror is the de-facto solution we're using right now, but if we get to a point where the traffic necessitates it, we'll look into a larger mirror network.

**Comment by praiskup** [comment]
> The Facebook mirror is the de-facto solution we're using right now

How do you configure the repositories?  Using two baseurl= options?

**Comment by sgallagher** [comment]
@davide125 Can you answer @praiskup here?

**Comment by michel-slm** [comment]
> How do you configure the repositories? Using two baseurl= options?

Yes; see https://dnf.readthedocs.io/en/latest/conf_ref.html?highlight=baseurl#baseurl-label

List the URLs in order of the one you want to try first (the Facebook mirror). one URL per line.

**Comment by yselkowitz** [comment]
ELN is now delivered through mirrors just like Fedora.
